docs-src/go/how-to-pause-a-schedule-in-go.md

```diff
@@ -0,0 +1,34 @@
+---
+id: how-to-pause-a-schedule-in-go
+title: How to pause a Schedule in Go
+sidebar_label: Pause Schedule
+description: Show how to unpause and pause a Schedule in Go.
+---
+
+`Pause` and `Unpause` enable the start or stop of all future Workflow Runs on a given Schedule.
+
+Pausing a Schedule halts all future Workflow Runs.
+Pausing can be enabled by setting `State.Paused` to `true`, or by using `Pause()` on the ScheduleHandle.
```

Comment:
nit: The word `enabled` normally means the feature can be used. I could see someone reading this getting and confused, thinking they need to change the `State.Paused` setting to allow them to call `Pause()` in the future. I would reword this to how you worded `Unpause()`